test/NLightning.Bolts.Tests/BOLT10/DNSBootstrapTests.cs

```diff
@@ -0,0 +1,48 @@
+using System.Net;
+using NLightning.Bolts.BOLT10.Services;
+using NLightning.Bolts.Tests.Utils;
+using ServiceStack.Text;
+using Xunit.Abstractions;
+
+namespace NLightning.Bolts.Tests.BOLT10;
+
+/// <summary>
+/// This tests do not run on Github CI for some reason DNS will not resolve
+/// </summary>
+public class DnsBootstrapTests
+{
+
+    public DnsBootstrapTests(ITestOutputHelper output)
+    {
+        Console.SetOut(new TestOutputWriter(output));
+    }
+
+    [Theory]
+    [InlineData(true)]
+    [InlineData(false, Skip = "Github doesn't allow UDP")]
```

Comment:
Instead of skipping everywhere, we can skip only if it's running in CI:

```
if (!tcpOnly && Environment.GetEnvironmentVariable("CI") == "true")
{
    return; // Skip this test on GitHub Actions if using UDP
}
```